### DISCUSSION TITLE
unclear OIDC_USERS_DB_URI usage

### DISCUSSION THREAD

**Discussion by OliverKleinBST** [mlflow-oidc/mlflow-oidc-auth#38] [Q&A] [answered]
I am wondering if OIDC_USERS_DB_URI could be the same as the one I use for "--backend-store-uri".
When do use the same I currently get some alembic error, so I am wondering if its intended to be the same at all, as we have similar tables about the users and experiements.

**Comment by kharkevich** [comment, marked as answer]
MLFlow and the OIDC Auth plugin cannot share the same database or schema, but they can use the same database server.
It's recommended to use separate databases for experiment tracking and for managing users and permissions.
